packages/suite/src/actions/suite/desktopUpdateActions.ts

```diff
@@ -81,7 +82,8 @@ export const ready = (info: UpdateInfo) => (dispatch: Dispatch, getState: GetSta
 };
 
 export const installUpdate =
-    (shouldInstallOnQuit?: boolean) => (_: Dispatch, getState: GetState) => {
```

Comment:
Boolean unnamed props makes code hard to read. Also I am not really happy about default values here. Especially when there is inverted (negated) logic.